### DISCUSSION TITLE
[FEATURE REQ] wizard/checker on regexp

### DISCUSSION THREAD

**Discussion by buzzqw** [autobrr/autobrr#1654] [Ideas]
Autobrr could build the regexp for tv series given some input from user
like series name and if given season , releaser, extension...
user can supply a file name already downloaded to check if parsing is ok

**Comment by zze0s** [comment]
Use the correct fields for shows, quality, season and ep etc instead. The built in release parser already translates from the release name and announce to those fields in the filter.

With **Match releases** you are on your own to make sure the regex works. We recommend [regex101.com](https://regex101.com) as a regex playground since they have a specific Golang implementation.

We do however plan do make it easier to test announces and expose the release parser so you can submit tests and see what it spits out.

**Comment by buzzqw** [comment]
"We do however plan do make it easier to test announces and expose the release parser so you can submit tests and see what it spits out." that would be very useful!

thanks
BHH
